portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body id="ravenclaw_stripes">
+    <div id="content">
+        <h1 id="title"><strong>Alisa's Gallery</strong></h1>
+        <div id="section">
+            <h2>Gallery</h2>
+            <div id="images">
+                <p><strong>My Taekwondo Team</strong></p>
+                <div id="image_label">
+                    <a href="/images/TKDCornellTeam.jpg">
+                        <img src="/images/TKDCornellTeam.jpg" width=250 alt="Cornell TKD Team Picture">
+                    </a>
+                    <p>This is my Taekwondo team!</p>
+                </div>
+                <div id="image_label">
+                    <a href="/images/BlueBeltGroup.jpeg">
+                        <img src="/images/BlueBeltGroup.jpeg" width=250 alt="Cornell Blue Belts Picture">
+                    </a>
+                    <p>These are my fellow blue belts!</p>
+                </div>
+                <p><strong>Fun Memories</strong></p>
+                <div id="image_label">
```

Comment:
instead of id, do you want to use class instead? id should be unique per html element

Reply:
Added a commit which uses class for multiple html elements and id for unique html elements.